stat/fetcher.go

```
@@ -109,7 +109,9 @@ func (self *Fetcher) SetBlockchain(blockchain Blockchain) {
 
 func (self *Fetcher) Run() error {
 	log.Printf("Fetcher runner is starting...")
-	self.runner.Start()
+	if err := self.runner.Start(); err != nil {
+		log.Fatalf("Runner start error: %s", err.Error())
```

you should return the error here.